flake.nix

```diff
@@ -85,9 +85,11 @@
           #  - run integ tests
           #  - do common development operations (e.g. lint, debug, and manage repos)
           inherit system;
-          packages = tls_packages ++ common_packages;
-          # This env var can be over-ridden instead of recreating the shellHook.
```

Comment:
Can we add we comment here ? Like the one that was removed.

Reply:
This wasn't right- the shellHook wasn't picking up the updated env var.